doc/api/crypto.markdown

```diff
@@ -727,6 +742,15 @@ treated as the key with no passphrase and will use `RSA_PKCS1_OAEP_PADDING`.
 
 NOTE: All paddings are defined in `constants` module.
 
+## crypto.timingSafeEqual(a, b)
+
+Returns true if a is equal to b, without leaking timing information that would
+help an attacker guess one of the values. This is suitable for comparing secret
+values other than HMACs, such as authentication cookies or
```

Comment:
Fixed to just say "secret values like authentication cookies and capability urls." I wrote this comment thinking people would be reading it in comparison to `Hmac.validate`, but of course they are most likely to read this comment on its own, so the comparative language is just confusing.